tests/test_custodia.py

```diff
@@ -32,7 +32,7 @@ def find_port(host='localhost'):
 [global]
 server_version = "Secret/0.0.7"
 server_url = ${SOCKET_URL}
-auditlog = test_audit.log
+auditlog = tests/tmp/test_audit.log
```

Comment:
if we are going to add directory structure then it should be passed in as ${TESTS_DIR} like we do for SOCKET_URL